### ISSUE TITLE
turn MNE-BIDS pipeline into a BIDS app

### ISSUE THREAD

**Issue by Remi-Gau** [mne-tools/mne-bids-pipeline#1007]
Random question: has there been attempts to turn this pipeline into a full BIDS app?

**Comment by hoechenberger** [comment]
I would need to look up again what specifically defines a BIDS app, but if it's about Docker'izing the whole thing, then I've got something that's almost ready to share actually ...

**Comment by Remi-Gau** [comment]
yup it's mostly defined by containerisation + some requirements on the CLI

See the FAQ for BIDS apps developers : https://bids-website.readthedocs.io/en/latest/faq/bids-apps.html#for-developers
and the bids apps spec: https://bids-standard.github.io/execution-spec/

FYI: we are in the process of reorganizing the different BIDS websites, so defo let me know if something is unclear or hard to find: would be a good pointer on how to reduce the friction.

**Comment by hoechenberger** [comment]
I think I can do that, yes :)
